docs/source/development.rst

```diff
@@ -0,0 +1,104 @@
+.. _development-policy:
+
+==================
+Development policy
+==================
+
+In this section, we describe the development policy the core team of Chainer Chemistry follows.
+Developers who are thinking to send PRs to the repository are encouraged to read the following sections
+before starting implementation.
+
+
+Versioning policy
+=================
+
+Basically, we follow the `semantic versioning v2.0.0 <https://semver.org>`_.
+In Chainer Chemistry, *public APIs* in the sense of semantic versioning are ones in `the document <http://chainer-chemistry.readthedocs.io/en/latest/index.html>`_.
+
+We follow these rules about versioning during the major version zero in addition to ones described in the the semantic versioning:
+
+* We do not plan any scheduled releases.
+* We do not plan any pre releases.
+* We release the minor version when the core development team agrees. Typically, we do so when (1) sufficient number of features are added since the last minor release (2) the latest release cannot run the example code in the master branch of the repository (3) critical bugs are found. But we are not restricted to them.
+* If we find critical bugs, we should release a patch version or a minor version that fixes them. The core development team will determine which version to release.
+
+We do not have a concrete plan about versioning strategy after v1.0.0.
+
+
+Compatibiity policy
+===================
+
+As an immediate consequence of the semantic versioning, we may break compatibility of public APIs including addition, deletion, change in semantics of them anytime in the major version zero.
+Since APIs of Chainer Chemistry are still immature and unstable, introduction of new features can sometime involve compatibility break.
+If we are faced with a dilemma between cost for backward compatibility and benefit of new features, we are likely to give up the former because we want to place importance on introducing new features as early as possible. Of course, we care backward compatibility whenever it is easy and low-cost.
```

Comment:
whenever it is easy and low-cost -> whenever possible

Reply:
It slightly changes the meaning: how do we treat when it is possible but takes much cost.